DEVELOPMENT.md

```diff
@@ -0,0 +1,142 @@
+# How-To Compile, Install & Run Jellyfin Template plugin
+  
+  
+document creation date: 01/01/2021
+  
+## Intro
+  
+1. Jellyfin is a Plex OpenSource equivalent, to read more about it goto: https://jellyfin.org  
```

Comment:
```suggestion
1. Jellyfin is an open source media server, to read more about it, go to: https://jellyfin.org  
```

Don't think we really need the plex comparison if someone is attempting to make a jellyfin plugin